tests/functional/002-expected-fail/functional/source/data/config.json

```diff
@@ -0,0 +1,3 @@
+{
+    "code": 2
```

Comment:
expected fail je vzdycky code:1 https://developers.keboola.com/extend/common-interface/environment/#return-values

na funkci to nema vliv, ale trosku me to zmatlo

Reply:
Done. True.